helm/alfresco-content-services/values.yaml

```diff
@@ -890,3 +896,16 @@ global:
 #   region:
 #   s3Bucket:
 #   comprehendRoleARN:
+
+  # -- Shared connections details for Elasticsearch/Opensearch, required when alfresco-elasticsearch-connector.enabled is true
+  elasticsearch:
```

Comment:
It's really not good that we have elasticseatch values in 3 places. 901line 736 and 88, but don't know how to do it better :(

Reply:
736 refers to the bitnami elasticsearch chart configuration, while the others are for how helm globals works (they need to be declared in each chart)